src/types/scanFile.ts

```diff
@@ -10,6 +10,7 @@ export namespace ScanFile {
     detectionRuleUUIDs?: string[]
     detectionRules?: Detector.Rule[]
     webhookURL: string
```

Comment:
Does this need to be made optional now that webhook goes into the `alertConfig`? I would also add a comment to inform devs that this field is deprecated and that they should use `alertConfig`. You could also just remove this altogether since you are going to release a new version of the package anyway.